cms/envs/test.py

```diff
@@ -132,6 +132,7 @@
 # This hack disables migrations during tests. We want to create tables directly from the models for speed.
 # See https://groups.google.com/d/msg/django-developers/PWPj3etj3-U/kCl6pMsQYYoJ.
 MIGRATION_MODULES = {app: "app.migrations_not_used_in_tests" for app in INSTALLED_APPS}
+MIGRATION_MODULES['credentials'] = 'app.migrations_not_used_in_tests'
```

Comment:
yes it needs to be solved for other apps as well. `credentials ` is in `INSTALLED_APPS ` but I guess migrations are not prevented for the apps that are added as `openedx.core.djangoapps.credentials`